src/ui/history_view.rs

```diff
@@ -128,11 +156,22 @@ pub fn history_view(
 	let dates_list: im::Vector<(usize, u64)> = dates.into();
 	let (dates_list, _set_dates_list) = create_signal(dates_list);
 
+	let db_height = db.clone();
+
 	let history_view = h_stack((
 		scroll(
 			virtual_stack(
 				VirtualDirection::Vertical,
-				VirtualItemSize::Fixed(Box::new(|| HISTORY_LINE_HEIGHT)),
+				VirtualItemSize::Fn(Box::new(move |_| {
+					if matches!(
+						db_height.get_field_kind(&id, &field),
+						DynFieldKind::MultiLine | DynFieldKind::MultiLineSecret
+					) {
+						MULTILINE_HEIGHT
+					} else {
+						HISTORY_LINE_HEIGHT
+					}
+				})),
```

Comment:
And this is how you make the `virtual_stack` dynamic height... you still have to tell the component but luckily we know